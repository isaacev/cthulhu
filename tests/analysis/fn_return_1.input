fn main() -> ::kernel::Num {
  let x = 123;
  x;
}
