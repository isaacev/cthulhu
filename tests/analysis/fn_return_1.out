
  INCORRECT RETURN TYPE  -----------------------------------------------------

  Expected the function to return the type `Int`. Instead the function body
  returns the type `()`:

    : fn_return_1
  4 | }
    | ^

